src/components/TokenBalance.test.tsx

```diff
@@ -148,123 +491,12 @@ describe('FiatExchangeTokenBalance and HomeTokenBalance', () => {
-              address: '0x00400FcbF0816bebB94654259de7273f4A05c762',
-              networkId: NetworkId['celo-alfajores'],
-              symbol: 'POOF',
-              balance: '5',
-              priceFetchedAt: Date.now(),
-            },
-            'celo-alfajores:0x10c892A6EC43a53E45D0B916B4b7D383B1b78C0F': {
-              priceUsd: '1.16',
-              address: '0x10c892A6EC43a53E45D0B916B4b7D383B1b78C0F',
-              tokenId: 'celo-alfajores:0x10c892A6EC43a53E45D0B916B4b7D383B1b78C0F',
-              networkId: NetworkId['celo-alfajores'],
-              symbol: 'cEUR',
-              balance: '7',
-              priceFetchedAt: Date.now(),
-            },
-          },
-        },
-      })
-
-      jest.mocked(getFeatureGate).mockReturnValue(true)
-
-      const { getByTestId } = render(
-        <Provider store={store}>
-          <TokenBalanceComponent />
-        </Provider>
-      )
-
       fireEvent.press(getByTestId('ViewBalances'))
       expect(navigate).toHaveBeenCalledWith(Screens.Assets)
     }
   )
 
-  it.each([HomeTokenBalance, FiatExchangeTokenBalance])(
-    'renders correctly with zero token balance and zero positions',
-    async (TokenBalanceComponent) => {
-      const store = createMockStore({
-        ...defaultStore,
-        tokens: {
-          tokenBalances: {},
-        },
-        positions: {
-          positions: [],
-        },
-      })
-
-      const tree = render(
-        <Provider store={store}>
-          <TokenBalanceComponent />
-        </Provider>
-      )
-
-      expect(tree.queryByTestId('ViewBalances')).toBeFalsy()
-      expect(getElementText(tree.getByTestId('TotalTokenBalance'))).toEqual('$0.00')
-    }
-  )
-
-  it('HomeTokenBalance shows View Assets link if balance is zero and feature gate is true', async () => {
-    const store = createMockStore({
-      ...defaultStore,
-      tokens: {
-        tokenBalances: {},
-      },
-      positions: {
-        positions: [],
-      },
-    })
-
-    jest.mocked(getFeatureGate).mockReturnValue(true)
-
-    const tree = render(
-      <Provider store={store}>
-        <HomeTokenBalance />
-      </Provider>
-    )
-
-    expect(tree.getByTestId('ViewBalances')).toBeTruthy()
-    expect(getElementText(tree.getByTestId('TotalTokenBalance'))).toEqual('$0.00')
-  })
-
-  it.each([HomeTokenBalance, FiatExchangeTokenBalance])(
-    'renders correctly with zero balance and some positions',
-    async (TokenBalanceComponent) => {
-      const store = createMockStore({
-        ...defaultStore,
-        tokens: {
-          tokenBalances: {},
-        },
-      })
-
-      const tree = render(
-        <Provider store={store}>
-          <TokenBalanceComponent />
-        </Provider>
-      )
-
-      expect(tree.queryByTestId('ViewBalances')).toBeFalsy()
-      expect(getElementText(tree.getByTestId('TotalTokenBalance'))).toEqual('$7.91')
-    }
-  )
-
-  it.each([HomeTokenBalance, FiatExchangeTokenBalance])(
+  it.each([HomeTokenBalance])(
```

Comment:
Since this applies to only home token balance, can we move this to its own
describe section? This would also allow us to change the `it.each` to a
`describe.each` so we don't end up adding tests in this section that
only applies to one of them?